apps/docs/pages/guides/platform/multi-factor-authentication.mdx

```diff
@@ -0,0 +1,42 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+import { IconCheck } from 'ui'
+
+export const meta = {
+  title: 'Multi-factor Authentication',
+  description: 'Enable multi-factor authentication (MFA) to keep your account secure.',
+  subtitle: 'Enable multi-factor authentication (MFA) to keep your account secure.',
+}
+
+Multi-factor authentication (MFA) adds an additional layer of security to your organization and prevents unauthorized access to your user account. MFA improves the security of your account by requiring a second factor to verify your user identity. Supabase allows users to enable MFA on their account and set it as a requirement for subsequent logins.
+
+## Supported Authentication Factors
+
+Currently, Supabase supports adding a unique time-based one-time password (TOTP) to your user account as an additional security factor. You can manage your TOTP factor using apps such as 1Password, Authy, Google Authenticator or Apple's Keychain.
+
+## Enable MFA
+
+You can enable MFA for your user account under your [Supabase account settings](/dashboard/account/security). Enabling MFA will result in all other user sessions to be automatically logged out and forced to sign-in again with MFA.
+
+<Admonition type="note">
+
+Supabase does not support returning recovery codes for MFA. Instead, we recommend that you register a backup TOTP factor to use in an event that you lose access to your primary TOTP factor.
+
+</Admonition>
+
+## Login with MFA
+
+Once you've enabled MFA for your Supabase user account, you will be prompted to enter your two-factor (2FA) authentication code. You can find your unique code generated by your preferred TOTP app.
+
+## Disable MFA
+
+You can disable MFA for your user account under your [Supabase account settings](/dashboard/account/security). On subsequent login attempts, you will not be prompted to enter a 2FA code.
+
+<Admonition type="caution">
+
+We strongly recommend that you do not disable MFA to avoid unauthorized access to your user account.
+
+</Admonition>
```

Comment:
Should we add a section on what to do in the event that they lose their backup MFA device?

(e.g. do reach out to support via the support form if you lose access to both your primary and backup MFA factors . Note that for security reasons, a MFA reset may take upwards of 3 days to process)

Reply:
yeah good point